### DISCUSSION TITLE
What version of Enterprise Server should I install?

### DISCUSSION THREAD

**Discussion by eamonburns** [community/community#136598] [Enterprise] [answered]
At [enterprise.github.com/releases](https://enterprise.github.com/releases), the latest version is 3.13.3, but at [enterprise.github.com/releases/3.13.3/download](https://enterprise.github.com/releases/3.13.3/download) there is a banner that says "This is not the latest release of GitHub Enterprise." which links to 3.14.0.rc1. This version _doesn't work_ because you can't update it (which turns out to be [a feature](https://docs.github.com/en/enterprise-server@3.14/admin/release-notes#3.14)).

So my questions are:

1. Why is there no warning on the [3.14 download page](https://enterprise.github.com/releases/3.14.0/download) about how it shouldn't be used in production.
2. What version should I _actually use_, seeing as there are warnings on every older version

(Context: We wasted around half a day setting up a server, just to find out that it was the release candidate, and that we will need to blow away the server and start again tomorrow with a useable version)

**Comment by pallsama** [comment, marked as answer]
Thank you for your feedback. I agree this is an issue. Between RC and GA of a release (3.14 now), we are showing 3.14 rc as the latest build and showing warnings in all other versions. We will fix this issue in near future. Currently, the latest version you can use on production instances is 3.13.3.
